xpk/xpk.py

```diff
@@ -607,17 +613,23 @@ def get_zone():
   return zone_outputs[-1]  # The zone name lives on the last line of the output
 
 
-def zone_to_region(zone) -> str:
```

Comment:
Why are we support zonal and regional clusters instead of just regional clusters? To me it adds complexity and removes sensible defaults. Is there a use case where we'd recommend the zonal cluster?

Reply:
v4 isn't available in a lot of the zones like `us-central2-a` etc. I repeatedly ran into errors when trying to regional cluster for v4. The default however is still a regional cluster, I haven't changed any defaults.

Reply:
Can we understand the root cause of the issues with the regional cluster? Not being available in a zone is no reason you shouldn't be able to make a regional cluster.

If we understand why this is necessary, we might decide we want to make this change. But I don't want to include  these zonal support without reason because it will permanently increase complexity.

Reply:
I think @Obliviour is better suited to answer this than me

Reply:
The issue we saw was with creating the cpu node pool that needs cpus from all zones in the region for regional clusters.

only a small portion of cpu types are available in all the zones in the region and there are some potential quota limits.

Reply:
I had a conversation with Victor and I don't see this change as important because there is a workaround (use the compatible CPU instances). Let's keep the instance flag but get rid of this one.

Reply:
removed zonal support flag!